bedrock/firefox/redirects.py

```diff
@@ -157,6 +157,7 @@ def firefox_channel(*args, **kwargs):
     redirect(r"^firefox/vpat(?:\.html)?", "http://website-archive.mozilla.org/www.mozilla.org/firefox_vpat/firefox-vpat-3.html"),
     # bug 1017564
     redirect(r"^mobile/.+/system-requirements/?$", "https://support.mozilla.org/kb/will-firefox-work-my-mobile-device"),
+    redirect(r"^firefox/firstrun/?$", "/firefox/new/"),
```

Comment:
Is this going to redirect every `/firstrun` URL? We do still have a /firstrun page for Dev Edition that still gets shown e.g. https://www.mozilla.org/en-US/firefox/119.0.a2/firstrun/

If we want to keep that (I'm not sure on the answer), then we might want to do a redirect in the view https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/views.py#L383